src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -1667,10 +1667,18 @@ private void ReportIfUnused(LocalFunctionSymbol symbol)
 
         public override BoundNode VisitLocal(BoundLocal node)
         {
+            LocalSymbol localSymbol = node.LocalSymbol;
```

Comment:
Is this related to target-typed conditional?

Reply:
Only tangentially, as I noticed a cascaded diagnostic in added tests.

---
In reply to: [410433785](https://github.com/dotnet/roslyn/pull/40604#discussion_r410433785) [](ancestors = 410433785)